### ISSUE TITLE
individuals_alive_at(0) raises IndexError

### ISSUE THREAD

**Issue by grahamgower** [tskit-dev/pyslim#93]
```
$ stdpopsim -e slim --slim-scaling-factor 10 HomSap 10 -l 0.01 -o /tmp/foo.trees
...
```
```python
>>> import pyslim
>>> pyslim.__version__
'0.402'
>>> ts = pyslim.load("/tmp/foo.trees")
>>> ts.individuals_alive_at(0)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/grg/src/pyslim/pyslim/slim_tree_sequence.py", line 567, in individuals_alive_at
    - self.individual(self.first_generation_individuals()[0]).time)
IndexError: index 0 is out of bounds for axis 0 with size 0
```

I guess the problem is that the tree sequence has been simplified, so it doesn't contain any first generation individuals. This used to work. Am I missing something?

**Comment by grahamgower** [comment]
I've changed my code to not rely on this (it was just in a test anyway, so no big deal). Feel free to close this if you don't think its worth fixing.

**Comment by petrelharp** [comment]
No, this is a bug! I'll investigate.

**Comment by petrelharp** [comment]
And `individuals_alive_at(0)` is an important method, I'll put out a new release for this fix. I guess I hadn't simplified any of my test examples, whoops.

**Comment by petrelharp** [comment]
ok, should be fixed in v0.403 (sigh)
